test/incompatibilities.js

```diff
@@ -127,4 +127,13 @@ suite("Parser", function() {
       new Shift.ArrayExpression([new Shift.LiteralNumericExpression(1), new Shift.LiteralNumericExpression(2)]), 
       new Shift.ExpressionStatement(new Shift.LiteralNumericExpression(0))));
   });
+
+  // ES5: allows only `\u Hex4Digits`
+  // ES6: allows `\u Hex4Digits` and `\u {HexDigits}`
```

Comment:
These tests shouldn't go into incompatibilities, since `\u{xxxx}` style escapes were syntax errors in ES5.